docs/sdk/features/edgedb.md

```diff
@@ -0,0 +1,60 @@
+# EdgeDB Feature
+
+## Overview
+
+The EdgeDB feature allows the saving of user data into our EdgeDB storage and to be able to use the data in segmenting decisions without needing
```

Comment:
maybe add a second sentence about how the data is stored and retrieved by user_id, allowing cross-platform data storage for the same user?